android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
Will pull your changes and check if it is. As far as the use is concerned, it totally boils down to the fact that whether the event has a lot of places to go. So when the organizer plans to include cafeteria, washrooms, and other markers it will clutter the map. See the above screenshot as an example.